src/app/static/src/app/store/adrUpload/adrUpload.ts

```diff
@@ -11,6 +11,8 @@ export interface ADRUploadState {
     uploadError: Error | null,
     uploading: boolean,
     uploadComplete: boolean,
+    releaseCreated: boolean,
+    releaseNotCreated: boolean,
```

Comment:
`releaseFailed` might be more appropriate. Let me know if you would like me to change.

Reply:
Yes, I prefer `releaseFailed`